src/middlewared/middlewared/plugins/nfs.py

```diff
@@ -564,6 +564,7 @@ async def pool_post_import(middleware, pool):
     path = f'/mnt/{pool["name"]}'
     for share in await middleware.call('sharing.nfs.query'):
         if any(filter(lambda x: x == path or x.startswith(f'{path}/'), share['paths'])):
+            await middleware.call('etc.generate', 'rc')
```

Comment:
Is this the right place to re-generate the rc.conf? Maybe we need to re-generate after bind ip change in NFS plugin?

Reply:
Nevermind, poor reading comprehension. Maybe also review whether it's also needed on nfs.update for bindip.